engines/wintermute/base/gfx/opengl/base_surface_opengl3d.cpp

```diff
@@ -188,6 +179,21 @@ bool BaseSurfaceOpenGL3D::create(int width, int height) {
 	return true;
 }
 
+bool BaseSurfaceOpenGL3D::putSurface(const Graphics::Surface &surface, bool hasAlpha) {
```

Comment:
hasAlpha is unused.

Reply:
Not sure about how to use it here or how this could be useful

Reply:
Fair enough.